### PR TITLE
PR: Add ellipsis to shorten notebook names

### LINKED ISSUE #66: Add ellipsis to shortened notebook names
I called this notebook "Hola mundo como vas el dia de hoy.ipynb" and this is how it's shown right now

seleccion_003

I propose

1. it should be shown as

        Hola mundo como ...

2. We should remove `.ipynb` from the notebook name. So instead of `foo.ipynb` we should show only `foo`. All notebooks have the `ipynb` extension, so there's no need to show it.
3. We should increase the max number of characters to show to **20** (instead of 15, the current limit).

### PR DESCRIPTION
**Pull request by dalthviz**
Fixes #66 

Preview:

imagen

### CODE REVIEW

File: spyder_notebook/widgets/client.py [hunk 1]
```diff
@@ -193,9 +194,10 @@ def get_filename(self):
 
     def get_short_name(self):
         """Get a short name for the notebook."""
-        sname = osp.basename(self.filename)
-        if len(sname) > 15:
-            sname = sname[:15]
+        sname = osp.splitext(osp.basename(self.filename))[0]
+        if len(sname) > 20:
+            fm = QFontMetrics(QFont())
+            sname = fm.elidedText(sname, Qt.ElideMiddle, 110)
```
Comment (ccordoba12) [inline spyder_notebook/widgets/client.py]: Let's change this to `ElideRight` instead of `ElideMiddle`

**Comment by ccordoba12** [comment]
@dalthviz, please update the screenshot so I can see how it looks now.

**Comment by dalthviz** [comment]
Preview: 

imagen

**Comment by ccordoba12** [comment]
Nice!! Thanks for this :-)

**Pull request merged**